packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1619,3 +1623,272 @@ Multiple IdP instances can be defined in a single Policy Action. This allows use
+| `activationRequirements`  | Contains a single Boolean property that indicates whether `emailVerification` should occur (`true`) or not (`false`, default).       | Object | Yes |        `false`                                                                                                                                                                                                              |
+| `preRegistrationInlineHooks` | (Optional) The `id` of at most one Registration Inline Hook                                                                       | Array   | No | N/A                                                                                                                                                                                                                        |
+| `profileAttributes.label`    | A display-friendly label for this property.                                                                                       | String  |  Required | N/A                                                                                                                                                                                                                      |
+| `profileAttributes.name`     | The name of a User Profile property. Can be an existing User Profile property.                                                   | String  |  Required | N/A                                                                                                                                                                                                                          |
+| `profileAttributes.required` | (Optional, default `FALSE`) Indicates if this property is required for enrollment                                                 | Boolean | Required | `FALSE`                                                                                                                                                                                                                        |
+| `profileAttributes` | A list of attributes for which to prompt the user during registration or progressive profiling. Where defined on the User schema, these attributes will be persisted in the User Profile. Non-schema attributes may also be added, which will not be persisted to the User's Profile, but will be included in requests to the Registration Inline Hook. A maximum of 10 Profile properties is supported.                                                         | Array | Required | N/A                                                                                                                                                                                                                        |
```

Comment:
should this also mention:
- also deals with app user schema?
- only allowed for READ-WRITE attributes (hidden and read only are not allowed)
- can also implicitly include any required User schema and AppUser schema?